openproblems/tasks/_batch_integration/batch_integration_feature/metrics/hvg_conservation.py

```diff
@@ -25,9 +25,20 @@
     image="openproblems-r-pytorch",
 )
 def hvg_conservation(adata):
+    from scanpy.pp import highly_variable_genes
     from scib.metrics import hvg_overlap
 
     adata_unint = adata.copy()
     adata_unint.X = adata_unint.layers["log_normalized"]
+    if "hvg_unint" not in adata_unint.uns:
```

Comment:
Why would it ever not be there? Just require it in the `check_dataset` and `check_method` APIs.